### PR TITLE
Fixed broken links

### PR DESCRIPTION
**Pull request by stevehoag**
Linking to MSDN as a workaround

### CODE REVIEW

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/standard/garbagecollection/implementing-dispose?branch=pr-en-us-1087](https://review.docs.microsoft.com/en-us/dotnet/articles/standard/garbagecollection/implementing-dispose?branch=pr-en-us-1087)  

**Status**: Succeeded.  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C9%5C28%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201609281833311318-1087%5Cworkflow_report.html).

**Comment by BillWagner** [comment]
LGTM. I checked the links and they are correct.
:shipit:

**Comment by stevehoag** [comment]
Thanks Bill 😃

File: docs/standard/garbagecollection/implementing-dispose.md [hunk 1]
```diff
@@ -410,7 +410,10 @@ Classes derived from the [System.Runtime.InteropServices.SafeHandle](xref:System
 
 * The [SafeMemoryMappedViewHandle](xref:Microsoft.Win32.SafeHandles.SafeMemoryMappedViewHandle) class, for memory views. 
 
-* The [SafeNCryptKeyHandle](xref:Microsoft.Win32.SafeHandles.SafeNCryptKeyHandle), [SafeNCryptProviderHandle](xref:Microsoft.Win32.SafeHandles.SafeNCryptProviderHandle), and [SafeNCryptSecretHandle](xref:Microsoft.Win32.SafeHandles.SafeNCryptSecretHandle) classes, for cryptography constructs. 
+* The [SafeNCryptKeyHandle](https://msdn.microsoft.com/en-us/library/microsoft.win32.safehandles.safencryptkeyhandle(v=vs.110).aspx), [SafeNCryptProviderHandle](https://msdn.microsoft.com/en-us/library/microsoft.win32.safehandles.safencryptproviderhandle(v=vs.110).aspx), and [SafeNCryptSecretHandle](https://msdn.microsoft.com/en-us/library/microsoft.win32.safehandles.safencryptsecrethandle(v=vs.110).aspx) classes, for cryptography constructs. 
+
+  > [!Note]
```
Comment (stevehoag) [inline docs/standard/garbagecollection/implementing-dispose.md]: I forgot that I had added this note. Technically it's still accurate given that the APIs can't be accessed from core until the build issue is resolved. I can remove it now, or we can wait until the APIs come back and remove it when I restore the old links.

Comment (mairaw) [inline docs/standard/garbagecollection/implementing-dispose.md]: I'll create a PR. Yes, you can access the APIs from Core, you can't access the docs for it.

**Pull request merged**